util/azure/utils.go

```diff
@@ -48,6 +51,14 @@ const (
 	ConnectedClusters           = "Microsoft.Kubernetes/connectedClusters"
 	OperationsEndpointFormatARC = "%s/providers/Microsoft.Kubernetes/operations?api-version=2021-10-01"
 	OperationsEndpointFormatAKS = "%s/providers/Microsoft.ContainerService/operations?api-version=2018-10-31"
+	retrieveServerApiError      = "unable to retrieve the complete list of server APIs"
+	apiserviceUnreachableError  = "the server is currently unable to handle the request"
+)
+
+var (
+	operationsMapLock = sync.RWMutex{}
+	operationsMap     = NewOperationsMap()
```

Comment:
is this expected? I observed operationsMap being used in range, lenght functions..